tests/cpp/cpp_unit/CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ add_executable(cpp_unit
         test_amqp.cpp
         test_server.cpp
         test_terminus.cpp
+        test_static_assert.c
```

Comment:
Can we please add this test as part of unit_test_SOURCES - https://github.com/skupperproject/skupper-router/blob/main/tests/CMakeLists.txt#L28

Reply:
yes